bullet_train-fields/app/helpers/fields/html_editor_helper.rb

```diff
@@ -5,7 +5,7 @@ def html_sanitize(string)
     return string unless string
     # TODO this is a hack to get around the fact that rails doesn't allow us to add any acceptable protocols.
     string = string.gsub("bullettrain://", TEMPORARY_REPLACEMENT)
-    string = sanitize(string, tags: %w[div br strong em b i del a h1 blockquote pre ul ol li], attributes: %w[href])
+    string = sanitize(string, tags: %w[div br strong em b i del a h1 blockquote pre ul ol li action-text-attachment figure figcaption img], attributes: %w[href sgid content-type url filename filesize width height presentation src class])
```

Comment:
I've added `class` here because the `action-text-attachment`, `figure`, and `figcaption` elements generated all include a `class`, so I figured it would be good to pass them through. Are there reasons we _shouldn't_ allow `class` in user-generated content?